### DISCUSSION TITLE
I need a mod to show progress bar at the back of taskbar icons

### DISCUSSION THREAD

**Discussion by bebotette** [ramensoftware/windhawk-mods#838] [Mod ideas]
I really don't like current progress indicator of icons in taskbar in win11. it's almost not visible and requires me a lot of guesswork and I have to manually check if a download is complete as an example.
I hope to see somebody create a mod to implement something like this:
![image](https://github.com/user-attachments/assets/84f60aa8-3457-4da4-aee4-21d3b7fc7b55)
(image not mine)

**Comment by m417z** [comment]
You might be able to achieve something similar with the [Windows 11 Taskbar Styler](https://windhawk.net/mods/windows-11-taskbar-styler) mod.

**Reply by bebotette** [reply]
not good with coding. Can you help?

**Reply by m417z** [reply]
No coding is required, but you'll have to spend some time experimenting with some targets and styles. UWPSpy can help.
I don't have time right now to look at it. You can also try asking here:
https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues
